### PR TITLE
Adding chapter 2

### PR DESCRIPTION
**Pull request by cansavvy**
In this pull request, I'm adding the second chapter solutions so we can review it as a group.

### CODE REVIEW

File: chapter-2.qmd [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+title: "Data visualization"
+---
+
+## Notes
+
+`alpha` aesthetic to add transparency to the filled density curves.
+
+This aesthetic takes values between 0 (completely transparent) and 1 (completely opaque)
+
+is to split your plot into **facets**, subplots that each display one subset of the data.
```
Comment (cansavvy) [inline chapter-2.qmd:11]: This seems like an incomplete sentence?

File: chapter-2.qmd [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+
+To facet your plot by a single variable, use [`facet_wrap()`](https://ggplot2.tidyverse.org/reference/facet_wrap.html). The first argument of [`facet_wrap()`](https://ggplot2.tidyverse.org/reference/facet_wrap.html) is a formula^[2](https://r4ds.hadley.nz/data-visualize.html#fn2)^, which you create with `~` followed by a variable name. The variable that you pass to [`facet_wrap()`](https://ggplot2.tidyverse.org/reference/facet_wrap.html) should be categorical.
+
+## Questions
+
+Make a scatterplot of `hwy` vs. `displ` using the `mpg` data frame. Next, map a third, numerical variable to `color`, then `size`, then both `color` and `size`, then `shape`. How do these aesthetics behave differently for categorical vs. numerical variables?
+
+that second part of the question is really unclear to me
+
+3.In the scatterplot of `hwy` vs. `displ`, what happens if you map a third variable to `linewidth`?
```
Comment (cansavvy) [inline chapter-2.qmd:21]:
```suggestion
3. In the scatterplot of `hwy` vs. `displ`, what happens if you map a third variable to `linewidth`?
```

File: chapter-2.qmd [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+
+i don't know how to map a third variable??
```
Comment (cansavvy) [inline chapter-2.qmd:23]:
```suggestion
TODO: i don't know how to map a third variable??
```
We should figure out how to do this.

**Review by cansavvy — COMMENTED** [review]
This was an example to run through what code review looks like sometimes. I haven't reviewed the full file.

File: chapter-2.qmd (hunk 1 above)
Comment (mine-cetinkaya-rundel) [reply chapter-2.qmd:11]: Since this is just in notes which won't be part of the final product, I'll resolve this comment without addressing it.

**Comment by mine-cetinkaya-rundel** [comment]
It looks like this chapter was already pushed to main with 1523f19 so I'm going to close this PR and edit the chapter on the main branch.

**Pull request closed without merging**